dashboard/src/components/HandleLink/HandleLink.tsx

```diff
@@ -0,0 +1,34 @@
+import { LinkComponentProps } from 'node_modules/@tanstack/react-router/dist/esm/link';
```

Comment:
`@types/react`  has `import { ComponentProps } from 'react';`
so you don't need to import stuff from node_modules 

also, every time you import a type use `import type`, we will be adding that eslint rule one day